src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,25 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+public class ByeCommand extends Command {
+
+    public ByeCommand() {
+        this.isBye = true;
+    }
```

Comment:
Any reason why the class and constructor do not have javadocs? I noticed the same thing in other files too.

Reply:
Have added the javadocs in recent commit!